src/ZEO/asyncio/client.py

```diff
@@ -547,7 +547,7 @@ def verify(self, server_tid):
     def get_peername(self):
         return self.protocol.get_peername()
 
-    def call_async_threadsafe(self, future, method, args):
+    def call_async_threadsafe(self, future, _, method, args):
```

Comment:
I also appreciate `_` for unused, but most linters I know will accept it as a prefix, i.e., `_some_unused`, which can meet both needs (indicating unused, while documenting).